weave-js/src/core/ops/primitives/date.test.ts

```diff
@@ -0,0 +1,96 @@
+import {constTimestamp} from '../../model';
+import {testClient} from '../../testUtil';
+import {opTimestampRelativeStringAutoFormat} from './date';
+import moment from 'moment';
+
+describe('date ops', () => {
+  describe('relativeStringAutoFormat', () => {
+    it('handles single unit pos', async () => {
+      const client = await testClient();
+      const timestamp1 = moment();
+      const timestamp2 = moment().add(moment.duration(1, 'days'));
+      const expr = opTimestampRelativeStringAutoFormat({
+        lhs: constTimestamp(timestamp2.valueOf()),
+        rhs: constTimestamp(timestamp1.valueOf()),
+      });
+      expect(await client.query(expr)).toEqual('1 day');
+    });
+
+    it('handles single unit neg', async () => {
+      const client = await testClient();
+      const timestamp1 = moment();
+      const timestamp2 = moment().add(moment.duration(1, 'days'));
+      const expr = opTimestampRelativeStringAutoFormat({
+        lhs: constTimestamp(timestamp1.valueOf()),
+        rhs: constTimestamp(timestamp2.valueOf()),
+      });
+      expect(await client.query(expr)).toEqual('-1 day');
+    });
+
+    it('handles year unit pos', async () => {
+      const client = await testClient();
+      const timestamp1 = moment();
+      const timestamp2 = moment().add(moment.duration(400, 'days'));
+      const expr = opTimestampRelativeStringAutoFormat({
+        lhs: constTimestamp(timestamp2.valueOf()),
+        rhs: constTimestamp(timestamp1.valueOf()),
+      });
+      expect(await client.query(expr)).toEqual('1.1 year');
```

Comment:
should be plural?

Reply:
had a poll with team and they agreed, will change